quasar-utils/javac-plugin/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junit_jupiter_version
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: junit_jupiter_version
+    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit_jupiter_version
+
+    if(JavaVersion.current() <= JavaVersion.VERSION_1_8) {
+        compileOnly files("${System.properties['java.home']}/../lib/tools.jar")
+        testRuntimeOnly files("${System.properties['java.home']}/../lib/tools.jar")
```

Comment:
How does this interact with Gradle's new [Java toolchains](https://docs.gradle.org/current/userguide/toolchains.html)?

Reply:
I'll investigate this

Reply:
[as I understand](https://blog.gradle.org/java-toolchains), toolchains are a feature only available since gradle 6.7, this sets of plugins currently builds on `5.6.4` (that set of dependencies is only required at plugin build time), this means I cannot do anything but ignore Gradle toolchains for now

Reply:
The Corda Gradle plugins for Corda 5.0 are currently targeting Gradle 6.6+, but there would be no problem raising this to Gradle 6.7+ if there were a good reason to use Toolchains. Unfortunately, you have created this unscheduled PR against a branch of the plugins that is still intended for use with Corda 4.x. So not only are you still using Gradle 5.6.4 but you are also missing all of the changes to `quasar-utils` to support the configurable `@Suspendable` annotation. (For reference, we did not make `co.paralleluniverse.fibers.SuspendExecution` throwables configurable in Quasar 0.8.x.)

I strongly recommend not continuing to work on this PR. As I understand it [CORE-187](https://r3-cev.atlassian.net/browse/CORE-187) was scheduled for "Milestone-2" anyway.

Reply:
I personally consider a mistake making `@Suspendable` configurable in the first place, but since we now have that, I thought it would be consistent to allow for a configurable `SuspendExecution` as well...

Reply:
Advertising functionality _which Quasar does not support_ is a waste of time and effort.

Please just stop this. You seem only to be implementing what you believe the requirements "should" be, and not what the requirements actually _are_. This is not productive.